src/components/forms/Form/ComplexForm.stories.tsx

```diff
@@ -295,12 +182,9 @@ const TemplateDialog = (args) => {
     <>
       <Button onPress={onPress}>Open Modal</Button>
       <DialogForm
+        title="Confirm delete"
         ref={deleteDeploymentFormRef}
-        onSubmit={(data) => console.log('Dialog Form Submit', data)}
-        submitProps={{
-          theme: 'danger',
-          label: 'Delete',
-        }}
+        submitProps={{ theme: 'danger', label: 'Delete' }}
```

Comment:
We set the danger theme via `danger` prop of the DialogForm. So this is correct, but not a recommended way.

Reply:
It's not my code, I didn't change any of this.

Reply:
I know, maybe it was me :)